drivers/bluetooth/hci/h4.c

```diff
@@ -499,7 +465,7 @@ static int h4_send(const struct device *dev, struct net_buf *buf)
 	const struct h4_config *cfg = dev->config;
 	struct h4_data *h4 = dev->data;
 
-	LOG_DBG("buf %p type %u len %u", buf, bt_buf_get_type(buf), buf->len);
+	LOG_DBG("buf %p type %u len %u", buf, buf->data[0], buf->len);
```

Comment:
It's quite implicit that the buffer type is in `data[0]`; should we have a comment here?

Reply:
It's a fair concern potentially in other places, but this is the H:4 driver which is explicitly specified to use this format, so it shouldn't really be anything special here IMO.